.travis.yml

```diff
@@ -1,17 +1,11 @@
 # custom
 language: python
-sudo: false
 python:
-    - 2.7
```

Comment:
If you're dropping 2.7, please mention that in the PR title.

(Also, I thought there were a few people who wanted to keep maintaining 2.7 support for ZF packages, and the consensus was we weren't going to make their life harder without a good reason?)